toolkits/landcover/test/int/dataset.int.test.js

```diff
@@ -99,7 +145,7 @@ withEarthEngine('Dataset', function() {
           expect(error).toBeUndefined();
```

Comment:
As your intent is to remove the actual collection from these tests,
maybe we should construct a fake collection with the real dates?